frontend/packages/console-shared/src/components/catalog/service/CatalogServiceProvider.tsx

```diff
@@ -47,7 +48,7 @@ const CatalogServiceProvider: React.FC<CatalogServiceProviderProps> = ({
     catalogBadgeProviderExtensions,
     extensionsResolved,
   ] = useCatalogExtensions(catalogId, catalogType);
-
+  const [disabledSubCatalogs] = useGetAllDisabledSubCatalogs();
```

Comment:
Isn't it easier to filter the items directly in `useCatalogExtensions` so that it returns already just an array of enabled `catalogTypeExtensions`?